server/models/userss.js

```diff
@@ -0,0 +1,45 @@
+import Authenticate from '../middle_ware/authenticate';
+
+
+const users = [
+    {
+        id: 1,
+        email: 'doyin@email.com',
+        firstName: 'Ade',
+        lastName: 'doyin',
+        password: Authenticate.hashPassword('password'),
+        address: 'cresent street',
+        status: 'verified',
+        isAdmin: false,
+        //token: Auth.generateToken({'doyin@email.com', false})
+    },
+    {
+        id: 2,
+        email: 'adebayo@ware.ng',
+        password: Authenticate.hashPassword('password'),
+        firstName: 'james',
+        lastName: 'john',
+        address: 'New york city',
+        status: 'not verified',
+        isAdmin: false,
+       // token: Auth.generateToken({'adebayo@ware.ng', false})
+    },
+    {
+        id: 3,
+        email: 'adebayo@quickcredit.com',
+        password: Authenticate.hashPassword('password'),
+        firstName: 'john',
+        lastName: 'doe',
+        address: 'Lekki phase I, Lagos',
+        status: 'verified',
+        isAdmin: true,
+      //  token: Auth.generateToken({'adebayo@quickcredit.com', true})
```

Comment:
Please remove all instances of dead code.

Reply:
#### What does this PR do?
fixes all dead codes
#### Description of Task to be completed?
upload to heroku
#### How should this be manually tested?
-cd into the quickcredit-App folder
-type npm run start-dev
-test heroku URL on postmann
#### Any background context you want to provide?

#### Screenshots (if appropriate)
-No screenshot
#### Questions:
-No question